alpaca/data/live/websocket.py

```diff
@@ -151,79 +167,91 @@ async def _consume(self) -> None:
+                return Orderbook(msg["S"], msg)
+            if msg_type in ("b", "u", "d"):
+                return Bar(msg["S"], msg)
+            if msg_type == "s":
+                return TradingStatus(msg["S"], msg)
+            if msg_type == "c":
+                return TradeCorrection(msg["S"], msg)
+            if msg_type == "x":
+                return TradeCancel(msg["S"], msg)
         return result
 
     async def _dispatch(self, msg: Dict) -> None:
-        """Distributes message from websocket connection to appropriate handler
+        """Distributes the message from websocket connection to the appropriate handler.
 
         Args:
             msg (Dict): The message from the websocket connection
         """
         msg_type = msg.get("T")
-        symbol = msg.get("S")
-        if msg_type == "t":
-            handler = self._handlers["trades"].get(
-                symbol, self._handlers["trades"].get("*", None)
-            )
-            if handler:
-                await handler(self._cast(msg_type, msg))
-        elif msg_type == "q":
-            handler = self._handlers["quotes"].get(
-                symbol, self._handlers["quotes"].get("*", None)
-            )
-            if handler:
-                await handler(self._cast(msg_type, msg))
-        elif msg_type == "b":
-            handler = self._handlers["bars"].get(
-                symbol, self._handlers["bars"].get("*", None)
-            )
-            if handler:
-                await handler(self._cast(msg_type, msg))
-        elif msg_type == "u":
-            handler = self._handlers["updatedBars"].get(
-                symbol, self._handlers["updatedBars"].get("*", None)
-            )
-            if handler:
-                await handler(self._cast(msg_type, msg))
-        elif msg_type == "d":
-            handler = self._handlers["dailyBars"].get(
-                symbol, self._handlers["dailyBars"].get("*", None)
-            )
-            if handler:
-                await handler(self._cast(msg_type, msg))
-        elif msg_type == "subscription":
-            sub = [f"{k}: {msg.get(k, [])}" for k in self._handlers]
+        if msg_type == "subscription":
+            sub = [f"{k}: {msg.get(k, [])}" for k in self._handlers if msg.get(k)]
             log.info(f'subscribed to {", ".join(sub)}')
-        elif msg_type == "error":
+            return
+
+        if msg_type == "error":
             log.error(f'error: {msg.get("msg")} ({msg.get("code")})')
+            return
+
+        if msg_type == "n":
+            symbols = msg.get("symbols")
+            for symbol in symbols:
+                handler = self._handlers["news"].get(symbol)
+                if handler:
+                    await handler(self._cast(msg))
+                    return
+            star_handler = self._handlers["news"].get("*")
+            if star_handler:
+                await star_handler(self._cast(msg))
+                return
```

Comment:
Please consider this implementation. What I thought was:
If we have multiple handler for a different symbols, a single news should be dispatched  into the symbol? Also, if there is no handler for a symbol, the star_handler should be called.

https://github.com/alpacahq/alpaca-py/pull/392/files#diff-f1f76d83a1c39c29072972280610b605f9e44c25ccf734321078c9e5b3f7cc38R235-R249